front/src/styles/scss/applications/operationalStudies/_scenario.scss

```diff
@@ -133,351 +152,428 @@
         gap: 0.5rem;
         margin-bottom: 0.5rem;
         height: 2rem;
+
         button {
           flex-grow: 1;
         }
       }
-      .filter-selector {
-        background-color: transparent;
-        border: none;
-        color: var(--secondary);
-        transition: 0.2s;
-        padding: 0.25rem 0.5rem;
-        border-radius: 4px;
-        &.on {
-          color: white;
-          background-color: var(--secondary);
-        }
-        &:hover {
-          background-color: var(--secondary);
-          color: var(--white);
-          .scenario-details-modify-button-text {
-            opacity: 1;
-          }
-        }
-      }
+
       .multiselect-delete {
         border: none;
         padding: 0.25rem 0.5rem;
-        border-radius: 4px;
+        border-radius: 0.25rem;
         border-color: transparent;
         background-color: rgba(var(--red-rgb), 0.75);
         color: var(--white);
+        height: 2rem;
         &.disabled {
           background-color: rgba(var(--secondary-rgb), 0.5);
         }
       }
+
       .multiselect-download {
         @extend .multiselect-delete;
         background-color: var(--primary);
       }
-      .scenario-timetable-toolbar {
-        display: flex;
-        align-items: center;
-        gap: 0.5rem;
-        margin-bottom: 0.5rem;
-        height: 2rem;
-        .multi-select-buttons {
+
+      .scenario-timetable-trains {
+        font-family: 'IBM Plex Sans';
+        background: linear-gradient(
+          to top,
+          var(--ambiantB15) calc(0.938rem),
+          rgba(0, 0, 0, 0.25) calc(1rem),
+          var(--ambiantB15) calc(1.063rem)
+        );
```

Comment:
I can't find any trace of this gradient in the mockup. It seems to be just a regular ambiantB15.

Reply:
Done